### ISSUE TITLE
issues displaying / printing certain Series objects

### ISSUE THREAD

**Issue by kellyjoy15** [Bears-R-Us/arkouda#1361]
The problem is at least with Categorical, Strings, boolean, and int64 types, at least in the cases below:
```python
akstrings = ak.array(['ab','ac','ab','da'])
x = ak.Categorical(akstrings)
ss = ak.Series(akstrings)
sc = ak.Series(x)
sb = ak.Series(ak.array([True,False,True,True]))
si = ak.Series(ak.array([1,2,3,4])
# all of the above works fine
print(ss)
# AttributeError: 'numpy.str_' object has no attribute 'to_ndarray'
print(sc)
# AttributeError: 'numpy.str_' object has no attribute 'to_ndarray'
print(sb)
# AttributeError: 'numpy.bool_' object has no attribute 'to_ndarray'
print(si)
# AttributeError: 'numpy.int64' object has no attribute 'to_ndarray'
```

**Comment by pierce314159** [comment]
Hi @kellyjoy15, I believe this will be resolved by #1363. In the meantime, something like this should work
```python
>>> akstrings = ak.array(['ab','ac','ab','da'])
>>> x = ak.Categorical(akstrings)
>>> ss = ak.Series((ak.arange(akstrings.size),akstrings))
>>> ss
0    ab
1    ac
2    ab
3    da
dtype: object

>>> sc = ak.Series((ak.arange(x.size),x))
>>> sc
0    ab
1    ac
2    ab
3    da
dtype: object

>>> sb = ak.Series((ak.arange(4),ak.array([True,False,True,True])))
>>> sb
0     True
1    False
2     True
3     True
dtype: bool

>>> si = ak.Series((ak.arange(4),ak.array([1,2,3,4])))
>>> si
0    1
1    2
2    3
3    4
dtype: int64
```

**Comment by pierce314159** [comment]
Closing since this request is tracked by https://github.com/Bears-R-Us/arkouda/issues/1363
